### PR TITLE
Door property (P126)

### PR DESCRIPTION
**Pull request by prabau**
Introducing the door space property (P126), but just calling it "door" for the main name for simplicity.  This matches the usage in the reference from McCartan in the README. (And "door" functions similarly to an adjective here.)

See #295.

Can someone check the MR for McCartan's paper?  This needs to be added in two places in P000126.md.

### CODE REVIEW

File: theorems/T000143.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if:
+  P000126: true
+then:
+  P000001: true
+refs:
+- mo: 405781
+  name: How do finite door spaces work?
+---
+
+If $x$ and $y$ are two topologically indistinguishable points, the singleton $\{x\}$ is neither open nor closed.
```
Comment (StevenClontz) [inline theorems/T000143.md:12]: Suggestion: either clarify that you're proceeding by contrapositive, or use this direct proof:

Given two distinct points $x,y$, either $\{x\}$ is an open neighborhood of $x$ not containing $y$, or $X\setminus\{x\}$ is an open neighborhood of $y$ not containing $x$; therefore the points are topologically distinct.

**Review by StevenClontz — CHANGES_REQUESTED** [review]
Just the one suggestion to consider before approval.

File: theorems/T000143.md (hunk 1 above)
Comment (StevenClontz) [reply theorems/T000143.md:12]: (Github ate my backslashes it seems.)

Comment (prabau) [reply theorems/T000143.md:12]: Yeah, we can clarify a bit.  Let me incorporate your change.

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**